package.json

```diff
@@ -59,15 +61,15 @@
     "react-test-renderer": "^16.0.0",
     "rimraf": "^2.6.2",
     "rollup": "^0.59.1",
-    "rollup-plugin-babel": "^3.0.4",
+    "rollup-plugin-babel": "^4.0.0-beta.4",
     "rollup-plugin-commonjs": "^9.1.3",
     "rollup-plugin-node-resolve": "^3.3.0",
     "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-size-snapshot": "^0.4.1",
-    "sinon": "^5.0.0",
-    "size-limit": "^0.18.0"
+    "sinon": "^5.0.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.0.0-beta.47",
```

Comment:
Would be better to match Material-UI's scope, beta 42 or something like that.

Reply:
Nope. Update to the latest version whenever possible. Small duplication in mismatched versions is ok. Will be fixed when babel7 will be released or mui will update Babel.

This is still better than inlining because the version of these plugins can be used in different package. So we still win.